docs/source/markdown/podman-logs.1.md

```diff
@@ -52,7 +52,7 @@ To view a container's logs:
 ```
 podman logs -t b3f2436bdb978c1d33b1387afb5d7ba7e3243ed2ce908db431ac0069da86cb45
 
-2017/08/07 10:16:21 Seeked /var/log/crio/pods/eb296bd56fab164d4d3cc46e5776b54414af3bf543d138746b25832c816b933b/c49f49788da14f776b7aa93fb97a2a71f9912f4e5a3e30397fca7dfe0ee0367b.log - &{Offset:0 Whence:0}
+2017/08/07 10:16:21 Sought /var/log/crio/pods/eb296bd56fab164d4d3cc46e5776b54414af3bf543d138746b25832c816b933b/c49f49788da14f776b7aa93fb97a2a71f9912f4e5a3e30397fca7dfe0ee0367b.log - &{Offset:0 Whence:0}
```

Comment:
the log message is generated by: https://github.com/hpcloud/tail/blob/master/tail.go#L244

Should we replace it?

Reply:
Nope should be reverted.